bin/MadGraph5_aMCatNLO/cards/production/13TeV/exo_diboson/Spin-0/Radion_VBF_ZZ_inclu/Radion_VBF_ZZ_inclu_narrow_M2500/Radion_VBF_ZZ_inclu_narrow_M2500_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+output Radion_VBF_ZZ_inclu_narrow_M2500 -nojpeg
+
+set complex_mass_scheme False
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model heft_radion
+define p = u c d s u~ c~ d~ s~
```

Comment:
there's an inconsistency here, p and j should normally have the same definition.  (The b should be included in both if and only if it is massless in the model)